classes/SuperDirt.sc

```diff
@@ -124,7 +124,7 @@ SuperDirt {
 		soundLibrary.loadOnly(names, path, appendToExisting )
 	}
 
-	loadSoundFileFolder { |folderPath, name, appendToExisting = false|
+	loadSoundFileFolder { |folderPath, name, appendToExisting = false, sortFiles = true|
 		soundLibrary.loadSoundFileFolder(folderPath, name, appendToExisting)
```

Comment:
you have to pass the argument on to `soundLibrary.loadSoundFileFolder` …

Reply:
Yes, you're right, added it.